kernel/src/main/scala/org/typelevel/sbt/TypelevelKernelPlugin.scala

```diff
@@ -26,12 +26,15 @@ object TypelevelKernelPlugin extends AutoPlugin {
 
   object autoImport {
     lazy val tlIsScala3 = settingKey[Boolean]("True if building with Scala 3")
-    def replaceCommandAlias(name: String, contents: String): Seq[Setting[State => State]] =
+
+    def tlReplaceCommandAlias(name: String, contents: String): Seq[Setting[State => State]] =
       Seq(GlobalScope / onLoad ~= { (f: State => State) =>
         f andThen { s: State =>
           BasicCommands.addAlias(BasicCommands.removeAlias(s, name), name, contents)
         }
       })
+
+    implicit def tlCommandOps(commands: List[String]): TlCommandOps = new TlCommandOps(commands)
```

Comment:
I know implicits aren't the most popular. Thoughts on this `mkCommand` syntax to help define the aliases?

Reply:
I don't think this one does enough to justify the magic.  If we frequently stubbed our toe quoting semi-colons, maybe, but we don't, and the helper doesn't fix that anyway.

Reply:
Fair enough, I'll scrap it.